state-chain/pallets/cf-threshold-signature/src/tests.rs

```diff
@@ -59,89 +81,81 @@ fn run_cfes_on_sc_events(cfes: &[MockCfe]) {
 	}
 }
 
-fn current_ceremony_id() -> CeremonyId {
-	<Test as crate::Config<Instance1>>::CeremonyIdProvider::get()
-}
-
 type ValidatorId = <Test as Chainflip>::ValidatorId;
 
 impl MockCfe {
 	fn process_event(&self, event: MockCfeEvent<ValidatorId>) {
-		match event {
-			MockCfeEvent::EthThresholdSignatureRequest(ThresholdSignatureRequest {
-				ceremony_id,
-				epoch_index: _,
-				key,
-				signatories,
-				payload,
-			}) => {
-				assert_eq!(key, current_agg_key());
-				assert_eq!(signatories, MockNominator::get_nominees().unwrap());
-
-				match &self.behaviour {
-					CfeBehaviour::Success => {
-						// Wrong request id is a no-op
-						assert_noop!(
-							EthereumThresholdSigner::signature_success(
-								RuntimeOrigin::none(),
-								ceremony_id + 1,
-								sign(payload)
-							),
-							Error::<Test, Instance1>::InvalidCeremonyId
-						);
-
-						assert_ok!(EthereumThresholdSigner::signature_success(
+		if let MockCfeEvent::EthThresholdSignatureRequest(ThresholdSignatureRequest {
+			ceremony_id,
+			epoch_index: _,
+			key,
+			signatories,
+			payload,
+		}) = event
+		{
+			//assert_eq!(key, current_agg_key());
```

Comment:
Let's re-add these asserts if they were there before

Reply:
These checks were removed because the previous behaviour didnt require responding to signature requests that were not the current key. However, new tests require this mock cfe to respond to keygen verification signatures which are not the current key. Therefore, these checks have to be removed.